modules/models/src/main/java/org/eclipse/xpanse/modules/models/common/exceptions/UnsupportedEnumValueException.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.common.exceptions;
+
+/**
+ * Exception thrown when there is an error during enum serialization.
+ */
+public class UnsupportedEnumValueException extends RuntimeException {
+
+    public UnsupportedEnumValueException(String message) {
+        super("Enums serialization error:" + message);
```

Comment:
No need of this extra text. Just pass the ''message" string.

Reply:
ok.